File: Programmers/Lv.1/K번째수/42748.js
function solution(array, commands) {
    const answer = [];
    
    for (let i=0; i<commands.length; i++) {
        const [s, e, idx] = commands[i];
        const arr = [...array.slice(s - 1, e)].sort((a, b) => a - b);
        answer.push(arr[idx - 1]);
    }
    
    return answer;
}

File: Programmers/Lv.1/K번째수/42748.java
import java.util.*;

class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];
        
        for (int i=0; i<commands.length; i++) {
            int s = commands[i][0];
            int e = commands[i][1];
            int idx = commands[i][2];
            
            int[] arr = Arrays.copyOfRange(array, s - 1, e);
            Arrays.sort(arr);
            
            answer[i] = arr[idx - 1];
        }
        
        return answer;
    }
}
